include/seqan3/range/views/minimiser.hpp

```diff
@@ -137,181 +225,153 @@ class minimiser_view : public std::ranges::view_interface<minimiser_view<urng_t>
      *
      * No-throw guarantee.
      */
-    auto end() const
+    sentinel end() const
     {
-        return std::default_sentinel;
-    }
-
-    //!\copydoc end()
-    auto cend() const
-    {
-        return end();
+        return {};
     }
     //!\}
 };
 
 //!\brief Iterator for calculating minimisers.
-template <std::ranges::view urng_t>
-template <typename rng_t>
-class minimiser_view<urng_t>::window_iterator
+template <std::ranges::view urng1_t, std::ranges::view urng2_t>
```

Comment:
We need to replace this with `bool` template parameters since our style guide changed. But this will be a follow up change and should not be part of this PR.